### PR TITLE
Use correct nonce to choose first ccm and improve certificate creation

### LINKED ISSUE #8200: CCMs chosen should be equal and greater to last ccm sent nonce
### Expected behavior

CCMs chosen should be equal and greater to last ccm sent nonce

### Actual behavior

CCMs chosen has nonce greater to last ccm sent nonce, so for the first scenario where initially we set lastSentCCM is set to `nonce: 0, height: 0` which will never make the first registrationCCM with nonce 0 to be included.

### Steps to reproduce

Run chain connector plugin and wait for the CCU to pick up the first CCU. It will not be able to include the first CCM.

### Which version(s) does this affect? (Environment, OS, etc...)

`v6.0.0-alpha.15`

### PR DESCRIPTION
**Pull request by ishantiw**
### What was the problem?

This PR resolves #8200 #8199 #8210 #8218 #8219

### How was it solved?

- Use `BigInt(-1)` as the initial lastSentCCM
- Improve certificate creation for the first certificate
- Take registrationHeight from the config that can be set by the user to identify the height from where we can start creating certificate and sending CCU

### How was it tested?

`npm t`

### CODE REVIEW

File: framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts [hunk 1]
```diff
@@ -104,6 +105,7 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 	private _ownChainID!: Buffer;
 	private _receivingChainID!: Buffer;
 	private _isReceivingChainMainchain!: boolean;
+	private registrationHeight!: number;
```
Comment (Phanco) [inline framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts:108]: `_ registrationHeight` if we follow the convention

**Comment by Phanco** [comment]
Wondering how this PR resolves #8199

**Comment by mitsuaki-u** [comment]
Code LGTM but one question: https://github.com/LiskHQ/lisk-sdk/issues/8199 says to "store registration ccm" but I don't see any storage added in this PR, only the height which is taken from config.

**Comment by ishantiw** [comment]
@Phanco @mitsuaki-u Related storing registration CCM, in the solution we can simply store the ccm height which can come from the config given by the user. So we don't need to unnecessarily store the whole registration CCM, here is the [related commit](https://github.com/LiskHQ/lisk-sdk/pull/8208/commits/a91db8635df93e4d597427df9d43bfb4454a41f4).

**Review by mitsuaki-u — APPROVED** [review]

**Review by Phanco — APPROVED** [review]

**Pull request merged**